open-api/rest-catalog-open-api.yaml

```diff
@@ -448,6 +476,35 @@ paths:
       summary: List all table identifiers underneath a given namespace
       description: Return all table identifiers under this namespace
       operationId: listTables
+      parameters:
+        - name: pageToken
```

Comment:
Can we define the `pageToken` and `pageSize` parameter in the common area and then just reference that as opposed to duplicating?

Reply:
+1

With this we can define some OpenAPI parameter descriptions like for `pageToken` we can set the minimum value to 1.

Reply:
Ack will fix this.